src/mame/psikyo/psikyo.cpp

```diff
@@ -1761,62 +1763,62 @@ Chips:  PS2001B
+	ROM_REGION( 0x020000, "audiocpu", 0 )       // Sound CPU Code
 	ROM_LOAD( "1-u63.bin", 0x00000, 0x20000, CRC(2025e387) SHA1(334b0eb3b416d46ccaadff3eee6f1abba63285fb) )
 
-	ROM_REGION( 0x001000, "mcu", 0 )       /* MCU */
-	ROM_LOAD( "4-u59.bin", 0x00000, 0x01000, NO_DUMP )
+	ROM_REGION( 0x001fff, "mcu", 0 )       // MCU, not hooked up
+	ROM_LOAD( "tengai_pic16c57_decap-c172_ncp.u59", 0x00000, 0x01fff, CRC(6c59dd4c) SHA1(d439ce2d00cefe400f0347d8ed1246c6f70ace5b) )
 
-	ROM_REGION( 0x600000, "gfx1", 0 )   /* Sprites */
+	ROM_REGION( 0x600000, "gfx1", 0 )   // Sprites
 	ROM_LOAD( "u20.bin",  0x000000, 0x200000, CRC(ed42ef73) SHA1(74693fcc83a2654ddb18fd513d528033863d6116) )
 	ROM_LOAD( "u22.bin",  0x200000, 0x200000, CRC(8d21caee) SHA1(2a68af8b2be2158dcb152c434e91a75871478d41) )
 	ROM_LOAD( "u21.bin",  0x400000, 0x200000, CRC(efe34eed) SHA1(7891495b443a5acc7b2f17fe694584f6cb0afacc) )
 
-	ROM_REGION( 0x400000, "gfx2", 0 )   /* Layer 0 + 1 */
-	ROM_LOAD( "u34.bin",  0x000000, 0x400000, CRC(2a2e2eeb) SHA1(f1d99353c0affc5c908985e6f2a5724e5223cccc) ) /* four banks of 0x100000 */
+	ROM_REGION( 0x400000, "gfx2", 0 )   // Layer 0 + 1
+	ROM_LOAD( "u34.bin",  0x000000, 0x400000, CRC(2a2e2eeb) SHA1(f1d99353c0affc5c908985e6f2a5724e5223cccc) ) // four banks of 0x100000
 
-	ROM_REGION( 0x400000, "ymf", 0 )    /* Samples */
-	ROM_LOAD( "u61.bin",  0x000000, 0x200000, CRC(a63633c5) SHA1(89e75a40518926ebcc7d88dea86c01ba0bb496e5) )    // 8 bit signed pcm (16KHz)
+	ROM_REGION( 0x400000, "ymf", 0 )    // Samples
+	ROM_LOAD( "u61.bin",  0x000000, 0x200000, CRC(a63633c5) SHA1(89e75a40518926ebcc7d88dea86c01ba0bb496e5) ) // 8 bit signed pcm (16KHz)
 	ROM_LOAD( "u62.bin",  0x200000, 0x200000, CRC(3ad0c357) SHA1(35f78cfa2eafa93ab96b24e336f569ee84af06b6) )
 
-	ROM_REGION16_LE( 0x040000, "spritelut", 0 )  /* Sprites LUT */
+	ROM_REGION16_LE( 0x040000, "spritelut", 0 )  // Sprites LUT
 	ROM_LOAD( "u1.bin",  0x000000, 0x040000, CRC(681d7d55) SHA1(b0b28471440d747adbc4d22d1918f89f6ede1615) )
 
 ROM_END
 
 ROM_START( tengaij )
-	ROM_REGION( 0x100000, "maincpu", 0 )        /* Main CPU Code */
+	ROM_REGION( 0x100000, "maincpu", 0 )        // Main CPU Code
 	ROM_LOAD32_WORD_SWAP( "2-u40.bin", 0x000000, 0x080000, CRC(ab6fe58a) SHA1(6687a3af192b3eab60d75ca286ebb8e0636297b5) ) // 1&0
 	ROM_LOAD32_WORD_SWAP( "3-u41.bin", 0x000002, 0x080000, CRC(02e42e39) SHA1(6cdb7b1cebab50c0a44cd60cd437f0e878ccac5c) ) // 3&2
 
-	ROM_REGION( 0x020000, "audiocpu", 0 )       /* Sound CPU Code */
+	ROM_REGION( 0x020000, "audiocpu", 0 )       // Sound CPU Code
 	ROM_LOAD( "1-u63.bin", 0x00000, 0x20000, CRC(2025e387) SHA1(334b0eb3b416d46ccaadff3eee6f1abba63285fb) )
 
-	ROM_REGION( 0x001000, "mcu", 0 )       /* MCU */
-	ROM_LOAD( "4-u59.bin", 0x00000, 0x01000, NO_DUMP )
+	ROM_REGION( 0x001fff, "mcu", 0 )       // MCU, not hooked up
+	ROM_LOAD( "tengai_pic16c57_decap-c172_ncp.u59", 0x00000, 0x01fff, CRC(6c59dd4c) SHA1(d439ce2d00cefe400f0347d8ed1246c6f70ace5b) ) // From a World PCB
```

Comment:
Is this like naming the pacman roms "pacman_2716_electronicdump-label.pcbloc"?

Reply:
I've renamed it as "label.socket", thanks.

![tengai_pic](https://user-images.githubusercontent.com/5889052/177529968-c78df407-c5e6-4d3a-b21e-c4671563ac2c.jpg)